content/pt/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,387 @@
+echo "if [ $commands[kubectl] ]; then source <(kubectl completion zsh); fi" >> ~/.zshrc # adicionar autocomplete permanentemente para o seu shell zsh
+```
+
+##  Contexto e configuração do Kubectl
+
+Defina com qual cluster do Kubernetes o `kubectl` se comunica e modifique os detales da configuração.
+Veja a documentação [Autenticando entre clusters com o kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) para
+informações detalhadas do arquivo de configuração.
+
+```bash
+kubectl config view # Mostrar configurações do kubeconfig mergeadas.
+
+# use vários arquivos kubeconfig ao mesmo tempo e visualize a configuração mergeada
+KUBECONFIG=~/.kube/config:~/.kube/kubconfig2 
+
+kubectl config view
+
+# obtenha a senha para o usuário e2e
+kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
+
+kubectl config view -o jsonpath='{.users[].name}'    # exibir o primeiro usuário
+kubectl config view -o jsonpath='{.users[*].name}'   # obtenha uma lista de usuários
+kubectl config get-contexts                          # exibir lista de contextos
+kubectl config current-context                       # exibir o contexto atual
+kubectl config use-context my-cluster-name           # defina o contexto padrão como my-cluster-name
+
+# adicione um novo cluster ao seu kubeconfig que suporte autenticação básica
+kubectl config set-credentials kubeuser/foo.kubernetes.com --username=kubeuser --password=kubepassword
+
+# salve o namespace permanentemente para todos os comandos subsequentes do kubectl nesse contexto.
+kubectl config set-context --current --namespace=ggckad-s2
+
+# defina um contexto utilizando um nome de usuário e o namespace.
+kubectl config set-context gce --user=cluster-admin --namespace=foo \
+  && kubectl config use-context gce
+ 
+kubectl config unset users.foo                       # excluir usuário foo
+```
+
+## Aplicar
+`apply` gerencia aplicativos através de arquivos que definem os recursos do Kubernetes. Ele cria e atualiza recursos em um cluster através da execução `kubectl apply`.
+Esta é a maneira recomendada de gerenciar aplicativos Kubernetes em ambiente de produção. Veja a [documentação do Kubectl](https://kubectl.docs.kubernetes.io).
+
+## Criando objetos
+
+Manifestos Kubernetes podem ser definidos em YAML ou JSON. A extensão do arquivo `.yaml`,
+`.yml`, e `.json` pode ser usado.
+
+```bash
+kubectl apply -f ./my-manifest.yaml            # criar recurso (s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml      # criar a partir de vários arquivos
+kubectl apply -f ./dir                         # criar recurso (s) em todos os arquivos de manifesto no diretório
+kubectl apply -f https://git.io/vPieo          # criar recurso (s) a partir de URL
+kubectl create deployment nginx --image=nginx  # iniciar uma única instância do nginx
+kubectl explain pods,svc                       # obtenha a documentação para pod e svc
```

Comment:
Can you please review this line, in the online doc it says:
"kubectl explain pods                           # get the documentation for pod manifests
"

Reply:
up